### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Tasks:
* [x] write user documentation in main README.md
  * [x] add documentation images
* [x] update asset JSON format documentation https://github.com/cyberbotics/robot-designer/blob/master/assets/README.md

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
I added the user manual documentation in the main README.md.
Another option is to create a Wiki Page (like https://github.com/cyberbotics/robot-designer/wiki/User-Manual).

Not sure what is the best solution.

File: README.md [hunk 1]
```diff
@@ -15,3 +19,103 @@ python -m SimpleHTTPServer
...
+The library contains all the available robot parts.
+The parts are grouped by robotics kit and the kit can be chosen using the combo box.
+Only parts that can be added to the current robot model are enabled.
+Parts that need a slot connection that it is currently not available with the robot model are shown in the library but they are disabled.
+
+
+## Part Viewer
+![Part Viewer](docs/images/user_manual_part_viewer.png)
+
+The part viewer shows the properties of the selected part and lets customize it
```
Comment (omichel) [inline README.md]:
```suggestion
The part viewer shows the properties of the selected part and lets the user customize it
```

File: README.md [hunk 2]
```diff
@@ -15,3 +19,103 @@ python -m SimpleHTTPServer
...
+
+You can delete parts by selecting them and clicking on the *Delete* button in the toolbar.
+Note that all the children parts connected to the selected part will also be deleted.
+
+## Interaction with the 3D Scene
+
+### Object Selection
+
+If you move your mouse pointer over a robot part, a white outline of the part is displayed.
+It is possible to select an object or a robot in the 3D scene by clicking on it using the left mouse button. Once selected, a blue outline is displayed.
```
Comment (omichel) [inline README.md]:
```suggestion
It is possible to select an object or a robot in the 3D scene by clicking on it using the left mouse button.
Once selected, a blue outline is displayed.
```

File: README.md [hunk 3]
```diff
@@ -15,3 +19,103 @@ python -m SimpleHTTPServer
...
+
+### Navigation in 3D
+
+#### Rotate Viewpoint
+
+To rotate the camera around the x and y axis, you have to set the mouse pointer in the 3D scene, press the left mouse button, and drag the mouse. The rotation is centered around the scene center.
```
Comment (omichel) [inline README.md]:
```suggestion
To rotate the camera around the x and y axis, you have to set the mouse pointer in the 3D scene, press the left mouse button, and drag the mouse.
The rotation is centered around the scene center.
```

**Review by omichel — CHANGES_REQUESTED** [review]
Very good. Thank you!

**Comment by stefaniapedrazzi** [comment]
:+1:

**Review by omichel — APPROVED** [review]

**Pull request merged**